arbor/benchmark_cell_group.cpp

```diff
@@ -39,7 +39,7 @@ benchmark_cell_group::benchmark_cell_group(const std::vector<cell_gid_type>& gid
         cg_targets.add_label(c.target, {0, 1});
     }
 
-    reset();
+    benchmark_cell_group::reset();
```

Comment:
Never knew this was ambiguous!

Reply:
Oh, it's not. _Here_. But assume you derive from `benchmark_cell_group` overriding the `reset` method.
Now, you'll **still** get the base class' `reset`. This makes it explicit. There's no fix except making the class
`final`.